vjtop/src/test/java/com/vip/vjtools/vjtop/util/FormatsTest.java

```diff
@@ -1,249 +1,124 @@
-        Assert.assertEquals("", retval);
-    }
-
-    @Test
-    public void joinInput1NullOutputNotNull() {
-
-        // Arrange
-        final ArrayList<String> list = new ArrayList<String>();
-        list.add("");
-        final String delim = null;
-
-        // Act
-        final String retval = Formats.join(list, delim);
-
-        // Assert result
-        Assert.assertEquals("", retval);
+    public void testFormatUsage() {
+        Assert.assertEquals("0m/0m", formatUsage(new Usage(1L, 2L, 2L)));
+        Assert.assertEquals("0m/0m/0m", formatUsage(new Usage(1L, 1L, 2L)));
     }
 
     @Test
-    public void leftStrInputNotNullNegativeOutputStringIndexOutOfBoundsException() {
-
-        // Arrange
-        final String str = "!";
-        final int length = -536_870_911;
-
-        // Act
-        thrown.expect(StringIndexOutOfBoundsException.class);
-        Formats.leftStr(str, length);
-
-        // Method is not expected to return due to exception thrown
+    public void testFormatUsageWithColor() {
+        Assert.assertEquals("0m/0m", formatUsageWithColor(new Usage(1L, 2L, 2L), new LongWarning()));
     }
 
     @Test
-    public void leftStrInputNotNullZeroOutputNotNull() {
-
-        // Arrange
-        final String str = "!!!!!!!!";
-        final int length = 0;
+    public void testJoin() {
+        final ArrayList<String> list = new ArrayList<String>();
+        final String delim = "AAAAAAAA";
 
-        // Act
-        final String retval = Formats.leftStr(str, length);
+        Assert.assertEquals("", join(list, delim));
 
-        // Assert result
-        Assert.assertEquals("", retval);
+        list.add("");
+        Assert.assertEquals("", join(list, delim));
     }
 
     @Test
-    public void rightStrInputNotNullNegativeOutputStringIndexOutOfBoundsException() {
-
-        // Arrange
-        final String str = "!!!!!!!!";
-        final int length = -1_048_568;
+    public void testLeftStr() {
+        Assert.assertEquals("", leftStr("!!!!!!!!", 0));
 
-        // Act
         thrown.expect(StringIndexOutOfBoundsException.class);
-        Formats.rightStr(str, length);
-
-        // Method is not expected to return due to exception thrown
-    }
-
-    @Test
-    public void rightStrInputNotNullPositiveOutputNotNull() {
-
-        // Arrange
-        final String str = "!!!!!!!!";
-        final int length = 2_147_221_512;
-
-        // Act
-        final String retval = Formats.rightStr(str, length);
-
-        // Assert result
-        Assert.assertEquals("!!!!!!!!", retval);
-    }
-
-    @Test
-    public void shortNameInputNotNullPositivePositiveOutputNotNull() {
-
-        // Arrange
-        final String str = "!";
-        final int length = 1;
-        final int rightLength = 6;
-
-        // Act
-        final String retval = Formats.shortName(str, length, rightLength);
-
-        // Assert result
-        Assert.assertEquals("!", retval);
+        leftStr("!", -536_870_911);
```

Comment:
I usually keep the `Method is not expected to return due to exception thrown` comment after the call to the object under test.

Reply:
I was in two minds about that myself. I checked a few recent PRs and saw that people had left out the comment so did the same in the name of minimisation. I'm open to adding it back in though.